tests-js/fido2.test.js

```diff
@@ -0,0 +1,35 @@
+import startFido2 from '../src/fido2.js'
```

Comment:
And we need to add the tests to travis.

Reply:
Tests (js) are in travis (runnig after code style check)